CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to OpenTracing
```

Comment:
nit: make this about the globaltracer specifically (that, or just do a redirect to the main OT-contrib guidelines.